test/e2e/functional/tests/e2e.go

```diff
@@ -209,12 +208,12 @@ var _ = Describe("metallb", func() {
   addresses:
   - 1.1.1.1-1.1.1.100
 `),
-			table.Entry("Test AddressPool object v1alpha1", "addresspool1", &metallbv1alpha1.AddressPool{
+			table.Entry("Test AddressPool object v1alpha1", "addresspool1", &metallbv1beta1.AddressPool{
```

Comment:
This change "breaks" the test + there are other places you changed the addresspools to v1beta - don't think it is necessary / fits in this PR

Reply:
Ah my bad, I copy-pasted too much. Thanks for spotting this.